tests/posix/common/src/mqueue.c

```diff
@@ -22,6 +22,7 @@ K_THREAD_STACK_ARRAY_DEFINE(stacks, N_THR, STACKSZ);
 
 char queue[16] = "server";
 char send_data[MESSAGE_SIZE] = "timed data send";
+char rec_data[MESSAGE_SIZE];
```

Comment:
> Ok, the docs for `KERNEL_COHERENCE` say that stack-based parameters are not valid in this case. I'd suggest updating the commit message to reference that variable and associated documentation so its clear that the ABI difference is documented.

If a respin happens, it might be good to add a 1-line comment referring to `KERNEL_COHERENCE` above this line for reference, and maybe mention that this could also be placed in stack memory if coherent memories are not required.

Reply:
Looks great, thanks!